cluster/examples/kubernetes/ceph/cluster-stretched.yaml

```
@@ -0,0 +1,56 @@
+#################################################################################################################
+# Define the settings for the rook-ceph cluster with common settings for a production cluster.
+# All nodes with available raw devices will be used for the Ceph cluster. At least three nodes are required
+# in this example. See the documentation for more details on storage settings available.
+
+# For example, to create the cluster:
+#   kubectl create -f common.yaml
+#   kubectl create -f operator.yaml
+#   kubectl create -f cluster-on-pvc.yaml
+#################################################################################################################
+
+apiVersion: ceph.rook.io/v1
+kind: CephCluster
+metadata:
+  name: rook-ceph
+  namespace: rook-ceph
+spec:
+  dataDirHostPath: /var/lib/rook
+  mon:
+    # Set the number of mons to be started. Must be an odd number, and is generally recommended to be 3.
+    count: 5
+    allowMultiplePerNode: true
+    stretchCluster:
+      # The ceph failure domain will be extracted from the label, which by default is the zone. The nodes running OSDs must have
+      # this label in order for the OSDs to be configured in the correct topology. For topology labels, see
+      # https://rook.github.io/docs/rook/master/ceph-cluster-crd.html#osd-topology.
+      failureDomainLabel: topology.kubernetes.io/zone
+      # The sub failure domaim is the secondary level at which the data will be placed to maintain data durability and availability.
+      # The default is "host", which means that each OSD must be on a different node and you would need at least two nodes per zone.
+      # If the subFailureDomain is set to "osd", the OSDs would be allowed anywhere in the same zone including on the same node.
+      # If set to "rack" or some other intermediate failure domain, those labels would also need to be set on the nodes where
+      # the osds are started.
+      subFailureDomain: osd
+      zones:
+      - name: a
+        arbiter: true
+      - name: b
+      - name: c
+  cephVersion:
+    # Stretch cluster support upstream is only planned starting in Ceph Octopus
+    image: ceph/daemon-base:latest-master
+    allowUnsupported: true
```

```suggestion
    allowUnsupported: false
```

Or just remove the line?